docs/whats-new/dotnet-7.md

```diff
@@ -0,0 +1,409 @@
+- [RuntimeInformation.OSArchitecture under emulation](../core/compatibility/interop/7.0/osarchitecture-emulation.md)
+- [Windows Forms obsoletions and warnings (.NET 7)](../core/compatibility/windows-forms/7.0/obsolete-apis.md)
+- [XmlSecureResolver is obsolete](../core/compatibility/xml/7.0/xmlsecureresolver-obsolete.md)
+- [Maximum precision for numeric format strings](../core/compatibility/core-libraries/7.0/max-precision-numeric-format-strings.md)
+- [Globalization APIs use ICU libraries on Windows Server](../core/compatibility/globalization/7.0/icu-globalization-api.md)
+- [BinaryFormatter serialization APIs produce compiler errors](../core/compatibility/core-libraries/7.0/binaryformatter-apis-produce-errors.md)
+- [All assemblies trimmed by default](../core/compatibility/deployment/7.0/trim-all-assemblies.md)
+- [Default authentication scheme](../core/compatibility/aspnet-core/7.0/default-authentication-scheme.md)
+- [Dynamic X509ChainPolicy verification time](../core/compatibility/cryptography/7.0/x509chainpolicy-verification-time.md)
+- [Middleware no longer defers to endpoint with null request delegate](../core/compatibility/aspnet-core/7.0/middleware-null-requestdelegate.md)
+- [Kestrel: Default HTTPS binding removed](../core/compatibility/aspnet-core/7.0/https-binding-kestrel.md)
+- [IHubClients and IHubCallerClients hide members](../core/compatibility/aspnet-core/7.0/ihubclients-ihubcallerclients.md)
+- [ContentRootPath for apps launched by Windows Shell](../core/compatibility/extensions/7.0/contentrootpath-hosted-app.md)
+- [x86 host path on 64-bit Windows](../core/compatibility/deployment/7.0/x86-host-path.md)
+- [MVC's detection of an empty body in model binding changed](../core/compatibility/aspnet-core/7.0/mvc-empty-body-model-binding.md)
+- [Event IDs for some Microsoft.AspNetCore.Mvc.Core log messages changed](../core/compatibility/aspnet-core/7.0/microsoft-aspnetcore-mvc-core-log-event-ids.md)
+- [Microsoft.Data.SqlClient updated to 4.0.1](../core/compatibility/aspnet-core/7.0/microsoft-data-sqlclient-updated-to-4-0-1.md)
+- [API controller actions try to infer parameters from DI](../core/compatibility/aspnet-core/7.0/api-controller-action-parameters-di.md)
+- [SignalR Hub methods try to resolve parameters from DI](../core/compatibility/aspnet-core/7.0/signalr-hub-method-parameters-di.md)
+- [Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv and libuv.dll removed](../core/compatibility/aspnet-core/7.0/libuv-transport-dll-removed.md)
+
+### Updated articles
+
+- [Breaking changes in .NET 6](../core/compatibility/6.0.md)
+  - Note break for .NET 6 servicing
+  - XmlSecureResolver breaking change
+  - x86 host path on x64/arm64
+- [DataContractSerializer retains sign when deserializing -0](../core/compatibility/serialization/7.0/datacontractserializer-negative-sign.md)
+  - Note break for .NET 6 servicing
+  - DataContractSerializer breaking change
+- [Breaking changes in .NET 7](../core/compatibility/7.0.md)
```

Comment:
This should be the only article listed in the "Breaking Changes" section. It links to all the others.